plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/internal/versions/VersionsPropertiesWriting.kt

```diff
@@ -10,8 +10,18 @@ import de.fayard.refreshVersions.core.internal.isAVersionAlias
 import de.fayard.refreshVersions.core.internal.versions.VersionsPropertiesModel.Companion.availableComment
 import de.fayard.refreshVersions.core.internal.versions.VersionsPropertiesModel.Section.Comment
 import de.fayard.refreshVersions.core.internal.versions.VersionsPropertiesModel.Section.VersionEntry
+import org.gradle.api.artifacts.ExternalDependency
 import java.io.File
 
+fun writeMissingEntriesInVersionProperties(newEntries: Map<String, ExternalDependency>) {
```

Comment:
Any reason this one is not `internal` @jmfayard?

Reply:
yes, it won't compile if it's internal. 
Feel free to improve it :)

Reply:
Then it should be annotated with `@InternalRefreshVersionsApi`. Ideally, it'd also not be top-level, but we already have a bunch of top level symbols in this case, so I'd revisit that latter point later.